hack/validate/default-seccomp

```diff
@@ -7,7 +7,7 @@ IFS=$'\n'
 files=( $(validate_diff --diff-filter=ACMR --name-only -- 'profiles/seccomp' || true) )
 unset IFS
 
-if [ ${#files[@]} -gt 0 ]; then
+if [ -n "${TEST_FORCE_VALIDATE}" ] || [ ${#files[@]} -gt 0 ]; then
```

Comment:
Should `${TEST_FORCE_VALIDATE}` be `${TEST_FORCE_VALIDATE:-}` here to ensure we don't get hung up by `set -u` if we run this script locally?